tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -1479,6 +1479,7 @@ process_params(void)
       }
     }
   }
+  lldbg_emit_lv_list(di);
```

Comment:
Current function processes constants (variables with parameter attribute). The call to lldbg_emit_lv_list should be at better place may be at the end of function process_formal_arguments. That place makes sure db->param_stack[] is populated.
It will also be good to make the call lldbg_emit_lv_list under the condition (db->routine_idx > 0) to make sure we have DISubprogram ready.

Reply:
Your comment leads to a significant change. But I like that change, I'll upload it later, when it passes tests.